oci/python/image.yaml

```
@@ -34,3 +34,14 @@ upload:
           - candidate
           - beta
           - edge
+  - source: canonical/chiselled-python
+    commit: f2b623d83eeb07882512821e0fd5ec703a602313
+    directory: python3.13/
+    release:
+      3.13-25.04:
+        end-of-life: "2026-01-31T00:00:00Z"
```

```suggestion
        end-of-life: "2026-01-15T00:00:00Z"
```
Source: https://git.launchpad.net/ubuntu/+source/distro-info-data/tree/ubuntu.csv?h=applied/ubuntu/noble-updates#n43